en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
-gui.xaero_hide_wp_coords=Hide Waypoint Coords
+gui.xaero_teleport_coordinates_hidden=Unable to teleport, because it could expose the waypoint coordinates. Disable the "Hide Waypoint Coordinates" option to be able to freely teleport again.
+gui.xaero_hide_wp_coords=Hide Waypoint Coordinates
 gui.xaero_waypoints_edit_show=Show
 gui.xaero_waypoints_edit_hide=Hide
 gui.xaero_teleport_anyway=Teleport Anyway
-gui.xaero_teleport_shows_coordinates=Shows you the coordinates!
+gui.xaero_teleport_shows_coordinates=It will expose the coordinates!
 gui.xaero_render_all_wp_sets=Render All Waypoint Sets
 gui.xaero_waypoints_global=Global
 gui.xaero_waypoints_local=Local
 gui.xaero_player_arrow_opacity=Arrow Opacity
 gui.xaero_box_waypoints_distance=Maximum render distance for local waypoints. Global waypoints are not affected.
 gui.xaero_connect_with_auto=Add Sub-World Connection
 gui.xaero_disconnect_from_auto=Delete Sub-World Connection
-gui.xaero_world_connection_tooltip=Connect/disconnect selected sub-world to/from the current "auto" one. Only connect sub-worlds that are from the same world save. For example, if a vanilla Nether portal takes you from sub-world A to sub-world B, then you can connect them. Sub-worlds connected to your current one are always at the top of the list and are marked by a * symbol.
+gui.xaero_world_connection_tooltip=Connect/disconnect selected sub-world to/from the current automatic one. Connect only sub-worlds that are from the same world save. For example, if a vanilla Nether portal takes you from sub-world A to sub-world B, then you can safely connect them. Other sub-worlds connected to the current one are always at the top of the list and are marked by an asterisk (symbol: *).
 gui.xaero_connect_with_auto_msg=Would you like to create the following connection between sub-worlds?
 gui.xaero_disconnect_from_auto_msg=Would you like to delete the following connection between sub-worlds?
-gui.xaero_teleport_not_connected=You are trying to teleport to a sub-world which isn't connected to the current "auto" one. If you are sure that this sub-world is from the same sub-server/world save, then you can enable teleportation by adding a connection in the waypoints menu -> Options -> Add Sub-World Connection. However, if you connect unrelated sub-worlds and teleport, then you might suffocate in a block or die from fall damage, so don't do that.
+gui.xaero_teleport_not_connected=You are trying to teleport to a sub-world that is not connected to the current automatic one. If you are sure that this sub-world is from the same world save as the automatic one, then you can enable teleportation by adding a connection on the waypoint screen -> Options -> Add Sub-World Connection. But beware, if you connect unrelated sub-worlds and teleport, then there is a good chance you will suffocate in a block or die from fall damage, so be careful and connect only the right ones.
```

Comment:
"from the same sub-server/world save" was better I think. I always found it difficult to explain what "the same world" is :S It's most often a sub-server, so it seems important to mention it. It's usually the same world save too but doesn't have to be. Doesn't really have to be the same sub-server either. Servers can do anything.

Reply:
Is it possible in Minecraft with Spigot/Paper to create multiple additional dimensions (each for every minigame) that behave exactly like the Overworld and are stored in the same world save? If not, then each sub-server has to use separate world saves.

Reply:
Yes, it is possible. You can make a plugin/mod that lets you create multiple dimensions on the server side that look exactly the same to the client. This isn't at all important here though. 
Each server obviously has a separate world save, whether what you asked is possible or not. I just think it's important to mention sub-servers so that it fits more cases. Having sub-servers on the same connection address is extremely common.

I think I confused you by going a bit off-topic in my initial reply without making it clear.
This line is about connecting different dimensions together, like Overworld and Nether. Usually, entering a Nether portal will take you to the Nether on the same server/save. However, you could also have a separate server for each dimension if you wanted to. So 2 connectable worlds can be from a different sub-server/save. This is why I find it difficult to explain what "the same world" is. Neither mine or yours is technically 100% correct, but I want it to be understandable to as many people as possible, by giving some examples.

Reply:
Wow, this is much more complex than I though. I agree with you that the description should be understandable to as many people as possible. Give me a while, I'm going to add "sub-server" here and in `gui.xaero_world_connection_tooltip` as well. What do you think?

Reply:
Not sure how I missed this one. Yes, that's sounds good to me, but don't change it for _gui.xaero_world_connection_tooltip_. That one seems fine.

Reply:
Already done.

Reply:
Great!